recipe/lib-avahi.sh

```diff
@@ -0,0 +1,23 @@
+set -xou
+
+NPROC=$CPU_COUNT
+if [[ $(uname) == "Darwin" ]]; then
+    export CFLAGS=-D__APPLE_USE_RFC_2292
+fi
+
+if [[ $(uname) == "Linux" ]]; then
+    NPROC=$(nproc)
+fi
+
+./configure --prefix $PREFIX \
+            --libdir ${PREFIX}/lib \
+            --bindir ${PREFIX}/bin \
+            --disable-qt5 \
+            --disable-gtk3 \
+            --disable-gdbm \
+            --disable-python \
+            --disable-mono
+
+make -j${NPROC}
+make install
+make distclean
```

Comment:
why do you run `distclean` here? conda-build directories are transient anyway.

Reply:
It seems that the build directories are recycled when building multiple outputs, at least that was the case on the CIs and the reason of why `distclean` is necessary

Reply:
Can you use an out-of-tree build to negate the need for this? the `python-avahi.sh` does this as an example.